src/crypto/crypto_timing.cc

```diff
@@ -17,6 +17,51 @@ using v8::Value;
 namespace crypto {
 namespace Timing {
 
+struct BigIntInfo {
+  int sign_bit;
+  int word_count;
+  std::unique_ptr<uint64_t> words;
```

Comment:
This seems wrong: This is not an array pointer – you might have meant `std::unique_ptr<uint64_t[]>` instead – but you’re using `Malloc` anyway, which is incompatible with either one of them.

Maybe `MallocedBuffer<uint64_t>` would make sense here? Or, otherwise, `std::vector<uint64_t>`?

Reply:
std::vector is better.